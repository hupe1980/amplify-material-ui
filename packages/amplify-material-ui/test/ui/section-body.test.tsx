import * as React from 'react';
import { render } from '@testing-library/react';
import { SectionBody } from '../../src';
import { withTheme } from '../helpers/withTheme';

describe('section-body', () => {
  it('should be rendered correctly', () => {
    const { asFragment } = render(
      withTheme(
        <SectionBody>
          <div />
        </SectionBody>,
      )
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
