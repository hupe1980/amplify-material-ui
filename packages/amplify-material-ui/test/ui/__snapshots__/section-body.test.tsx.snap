// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`section-body should be rendered correctly 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-box-1 MuiBox-root css-0"
  >
    <div />
  </div>
</DocumentFragment>
`;
